<launch>
    <arg name="camera" default="/rgb" />
    <arg name="image_topic" default="$(arg camera)/image_raw" />
    <arg name="camera_info_topic" default="$(arg camera)/camera_info" />
    <arg name="marker_length" default="0.1" />
    <arg name="camera_frame_id" default="rgb_camera_link" />
    <arg name="yaml_file" default="$(find aruco-tools)/config/ur3.yaml"/>
    <node name="storeTags" pkg="aruco-tools" type="storeTag.py" output="screen" args="$(arg image_topic) $(arg camera_info_topic) $(arg marker_length) $(arg camera_frame_id) $(arg yaml_file)"/>
</launch>